### PR TITLE
AS-11459 updated firmware docs to reflect changes in release 3.5

### PR DESCRIPTION
**Pull request by ramijebara**
Updated firmware documentation to reflect changes done in release 3.5

### CODE REVIEW

File: documentation/firmware-management/index.rst [hunk 1]
```diff
@@ -78,7 +77,7 @@ Hyperview uses the Firmware Version property (asset → *Information → Propert
 
 In both contexts, an asset is considered to have "managed firmware" only if the asset's firmware group is mapped in Hyperview Product Catalog. (A "firmware group" consists of firmware versions for a specific product model.)
```
Comment (rajanhv) [inline documentation/firmware-management/index.rst:78]: I suggest keep the term same. Either "Hyperview Catalog" or "Hyperview Product Catalog".

File: documentation/firmware-management/index.rst [hunk 2]
```diff
@@ -87,32 +86,29 @@ Upon discovery the Firmware Version property becomes read-only to prevent human
...
+As of Hyperview 3.4, any user can access Firmware Management features as long as you are using a Firmware Management-licensed instance (see :ref:`View license<View-license-doc>`.
 
 ====================================
 Navigating Firmware Management pages
 ====================================
 Hyperview features two main Firmware Management pages that can be considered "landing pages" for the feature set. These are:
 
-* The Firmware page (*Assets → Firmware*), which opens a grid of managed firmware that is grouped by manufacturer, asset type, and installed firmware version. It features a Details button at the row level to look up assets that have the same firmware version installed.
+* The Firmware page (*Assets → Firmware*), which opens a grid of managed and unmanaged firmware that is grouped by manufacturer, asset type, and installed firmware version. It features a Details button at the row level to look up assets that have the same firmware version installed.
 * The Information → Firmware page (asset → *Information → Firmware*), which opens a grid of installed and available firmware versions, as well as relevant action buttons.
```
Comment (rajanhv) [inline documentation/firmware-management/index.rst:97]: Use consistent capitalizations. "Asset" and "Firmware Versions" should be capitalized based on previous content.

**Review by rajanhv — COMMENTED** [review]
Overall looks good. Maintain consistency in terminology and capitalizations throughout the content. If choosing to capitalize a term, such as "Firmware Versions", then maintain throughout. If Catalog is referred to as a "Hyperview Catalog" then use that rather than "Hyperview Product Catalog".

File: documentation/firmware-management/index.rst (hunk 1 above)
Comment (ramijebara) [reply documentation/firmware-management/index.rst]: Agreed. I made the change.

**Review by rajanhv — APPROVED** [review]

**Pull request merged**